pinnwand/static/indent.js

```diff
@@ -0,0 +1,66 @@
+/// There should be a better way to handle the lexer
+/// I think :)
+
+
+function make_indent(event) {
+
+	var selector = document.querySelector("div.file-meta select[name='lexer']") // select the `select` under the file-meta div
```

Comment:
To get at the selector relevant to the textfield you should use ``event.target.parentNode.parentNode.querySelector("select[name='lexer']")`` so you always pick the ``select`` relevant to the current ``textarea``.

Reply:
Thanks.